bpf_helpers.h

```diff
@@ -0,0 +1,141 @@
+#ifndef __BPF_HELPERS_H
```

Comment:
Can't we use this header as provided by the kernel (as a package in the docker container)? 

http://lxr.free-electrons.com/source/samples/bpf/bpf_helpers.h